Components/Badge/Badge.css

```diff
@@ -0,0 +1,30 @@
+@import url(../Avatar/Avatar.css);
```

Comment:
Use of only small case letters is a good practice in naming files for a website. More details can be searched online regarding 'file naming conventions' for web developers : )